src/components/screens/ServiceTile.tsx

```diff
@@ -115,6 +144,8 @@ export const ServiceTile = ({
               primary={!hasDiagonalGradientBackground}
               center
               accessibilityLabel={`(${item.title}) ${consts.a11yLabel.button}`}
+              numberOfLines={numberOfLines}
+              style={hasFontStyle && fontStyle}
```

Comment:
why do you check if the object `fontStyle` has values? `style={}` for `fontStyle = {}` should work with applying no styles.

Reply:
Yes, it makes sense. You're right. We already create an empty object by default.

Reply:
[9fa123f](https://github.com/smart-village-solutions/smart-village-app-app/commit/9fa123f31738e43ad010657b72c7bd4219f63428)